openpgp/end_to_end_test.go

```diff
@@ -252,9 +252,9 @@ func encryptDecryptTest(t *testing.T, testSetFrom algorithmSet, testSetTo algori
 		t.Fatal("The message should be encrypted")
 	}
 	signKey, _ := signed.SigningKey(time.Now())
-	expectedKeyId := signKey.PublicKey.KeyId
-	if md.SignedByKeyId != expectedKeyId {
-		t.Fatalf("Message signed by wrong key id, got: %v, want: %v", *md.SignedBy, expectedKeyId)
+	expectedKeyID := signKey.PublicKey.KeyId
+	if md.SignedByKeyId != expectedKeyID {
+		t.Fatalf("Message signed by wrong key id, got: %v, want: %v", *md.SignedBy, expectedKeyID)
```

Comment:
I know the linter complains about this, but I think it's better to be consistent between `PublicKey.KeyId` and `expectedKeyId`, and since the former is a public API, I'd keep both as is.

Reply:
Yes, actually I'll revert all changes in `end_to_end_test` to avoid conflicts with #37